test/source/tests/gmail.ts

```diff
@@ -68,6 +68,13 @@ export const defineGmailTests = (testVariant: TestVariant, testWithBrowser: Test
       return gmailPage;
     };
 
+    const deleteMessage = async (gmailPage: ControllablePage) => {
+      await gmailPage.waitAndClick('[aria-label="More"]');
+      await gmailPage.press('ArrowDown', 5);
+      await gmailPage.press('Enter');
+      await Util.sleep(3);
+    };
```

Comment:
This deletes a message in a not very robust way - and not clear which message.

@rrrooommmaaa had a piece of code that deleted messages - I'll merge this and then make another issue about using that instead (it calls the Gmail API - more robust).